adagucserverEC/CNetCDFDataWriter.cpp

```diff
@@ -381,13 +381,20 @@ int CNetCDFDataWriter::init(CServerParams *srvParam, CDataSource *dataSource, in
           case CDF_UINT:
             ((unsigned int *)var->data)[j] = dimValue.toInt();
             break;
+          case CDF_INT64:
+            ((long *)var->data)[j] = dimValue.toLong();
+            break;
+          case CDF_UINT64:  // TODO: All unsigned versions don't work if the full unsigned range is needed
+            ((unsigned long *)var->data)[j] = dimValue.toLong();
+            break;
           case CDF_FLOAT:
             ((float *)var->data)[j] = dimValue.toFloat();
             break;
           case CDF_DOUBLE:
             ((double *)var->data)[j] = dimValue.toDouble();
             break;
           default:
+            CDBError("Unknown var type [%d]", var->getType());
```

Comment:
Maybe also log the dimension name (`dimName.c_str()`) so that you know what dimension the error relates to?

Reply:
Good idea!